app/controllers/v0/login_gov_callback_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module V0
+  class LoginGovCallbackController < ApplicationController
```

Comment:
Should we add this to the `SignIn` namespace and maybe create a new `Webhooks` namespace? Maybe idme or other services will have webhooks we can add in the future. Open to ideas but maybe something like:

```ruby
module SignIn
  module Webhooks
    class LogingovController
       
      def risc
        # ...
      end

    end 
  end
end